routes/lesson.js

```diff
@@ -11,6 +11,7 @@ const {
 router.use(authMiddleware)
 
 router.use(restrictTo(TUTOR))
+router.get('/', asyncWrapper(lessonController.getLessons))
```

Comment:
shouldn't it be above restrictTo middleware?

Reply:
but we talked that there will be lessons precisely (available and finished), didn't we?